tensorflow/java/maven/run_inside_container.sh

```diff
@@ -240,14 +279,17 @@ cd "${DIR}"
 # Comment lines out appropriately if debugging/tinkering with the release
 # process.
 # gnupg2 is required for signing
-apt-get -qq update && apt-get -qqq install -y gnupg2
+apt-get -qq update && apt-get -qqq install -y gnupg2 && apt-get -qqq install -y git
```

Comment:
Will a single `apt-get install` command suffice? As in: `apt-get -qqq install -y gnupg2 git`